src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcements.tsx

```diff
@@ -36,23 +36,34 @@ export const Announcements = ({style: containerStyle}: Props) => {
 
   return (
     <View style={containerStyle} testID="announcements">
-      <SectionHeading>{t(DashboardTexts.announcemens.header)}</SectionHeading>
-      <ScrollView>
-        {filteredAnnouncements.map((a, i) => (
+      <View style={style.headerWrapper}>
+        <SectionHeading>{t(DashboardTexts.announcemens.header)}</SectionHeading>
+      </View>
+      <ScrollView
+        contentContainerStyle={style.scrollView}
+        horizontal={filteredAnnouncements.length > 1}
+      >
+        {filteredAnnouncements.map((a) => (
           <Announcement
+            key={a.id}
             announcement={a}
-            style={i < filteredAnnouncements.length - 1 && style.announcement}
+            style={filteredAnnouncements.length > 1 && style.announcement}
           />
         ))}
       </ScrollView>
     </View>
   );
 };
 
-const useStyle = StyleSheet.createThemeHook((theme) => {
-  return {
-    announcement: {
-      marginBottom: theme.spacings.medium,
-    },
-  };
-});
+const useStyle = StyleSheet.createThemeHook((theme) => ({
+  headerWrapper: {
+    marginHorizontal: theme.spacings.medium,
+  },
+  scrollView: {
+    marginHorizontal: theme.spacings.medium,
```

Comment:
There is something iffy about using margins on `ScrollView`. And when testing this I see there is an issue with a missing right margin
<img width="552" alt="Screenshot 2023-12-21 at 11 36 30" src="https://github.com/AtB-AS/mittatb-app/assets/675421/92359339-6e3a-4262-ae35-6776b69314a4">

My tip is to try using `gap` and `paddingHorizontal` on the `ScrollView`, and remove the margin on the announcement.

Reply:
Thanks. I've noticed myself. I'll try your solution

Reply:
Fixed in https://github.com/AtB-AS/mittatb-app/pull/4148/commits/94bd79b35b70792058587fb648a613edbd4fe01b